src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -65,12 +69,9 @@ struct OperationParameters {
             methodDecoding.append(param)
         }
 
-        // Add a blank key,value in order for Stencil generates an empty dictionary for QueryParams and PathParams constructor
-        if query.required.isEmpty {
-            query.required.append(KeyValueViewModel(key: "", value: "\"\""))
-        }
+        // Add a blank key,value in order for Stencil generates an empty dictionary for PathParams constructor
         if path.isEmpty {
-            path.append(KeyValueViewModel(key: "", value: "\"\""))
+            path.append(KeyValueViewModel(key: "", value: "\"\"", path: "\"\""))
```

Comment:
This seems odd to me. I would expect this to generate something like `"".""` which is clearly not right.

Reply:
Reverted.